ors-engine/src/main/java/org/heigit/ors/routing/RoutingProfile.java

```diff
@@ -526,11 +523,9 @@ public void setSpeedups(GHRequest req, boolean useCH, boolean useCore, boolean u
             req.setAlgorithm(Parameters.Algorithms.DIJKSTRA_BI);
             req.setProfile(profileNameCH);
         }
-        if (useCore) {
-            // fallback to a core profile without turn costs if one is available
```

Comment:
Why is the comment omitted?

Reply:
You mean why I didn't delete it?

Reply:
Hm. I guess  the IntelliJ magic just ate it. Will add it back.

Reply:
Fixed!